lib/services/thinkingParser.js

```diff
@@ -34,17 +35,19 @@ var errors = require('../errors'),
  * @param {Object} result           Object representing the structured request.
  */
 
-function parseHumidity(fields, result, callback) {
+function parseHumidity(fields, result, timestamp, callback) {
     if (fields.length >= 5) {
         result.attributes.push({
             name: 'humidity',
             value: fields[3],
-            type: 'float'
+            type: 'float',
+            metadatas: timestamp
```

Comment:
The problem here is that this attribute was not defined by me but in the
NGSI specification. So Not To Change.
El 05/08/2015 10:25, "Germán Toro del Valle" <notifications@github.com>
escribió:

> In lib/services/thinkingParser.js
> <https://github.com/telefonicaid/iotagent-thinking-things/pull/34#discussion_r36279249>
> :
>
> >      if (fields.length >= 5) {
> >          result.attributes.push({
> >              name: 'humidity',
> >              value: fields[3],
> > -            type: 'float'
> > +            type: 'float',
> > +            metadatas: timestamp
>
> It is kind of a minor issue so I leave it a your discretion, but
> "metadatas" in plural felt weird to me when I read it... I would say there
> is no a plural form for "metadata"... :) The point is that there are a lot
> of places when you should change this if you consider it appropriate... :(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/telefonicaid/iotagent-thinking-things/pull/34/files#r36279249>
> .
>